src/aws-cur/cost-report.ts

```diff
@@ -0,0 +1,102 @@
+import { Stack, aws_cur, aws_iam, aws_s3 } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+
+export class ReportGranularity {
```

Comment:
Any reason not to put these in a separate file?

Reply:
They are only subtypes of the ReportProps, so I thought that makes sense.